### ISSUE TITLE
model.train() in test mode

### ISSUE THREAD

**Issue by kwea123** [callmeray/PointMVSNet#16]
In `test.py` you set the model to train mode
https://github.com/callmeray/PointMVSNet/blob/b6f131eb72cb6de79adc9ec02412234caae0d271/pointmvsnet/test.py#L58

Shouldn't that be `model.eval()`? It might have huge impact on the batchnorm...
